platforms/core-runtime/logging/src/main/java/org/gradle/internal/featurelifecycle/LoggingDeprecatedFeatureHandler.java

```diff
@@ -92,12 +92,18 @@ public void featureUsed(final DeprecatedFeatureUsage usage) {
                 @Override
                 public void execute(InternalProblemSpec builder) {
                     ProblemSpec problemSpec = builder
-                        .label(usage.formattedMessage())
+                        .label(usage.getSummary())
                         .documentedAt(usage.getDocumentationUrl());
                     addPossibleLocation(diagnostics, problemSpec);
                     // TODO (donat) we can further categorize deprecation warnings https://github.com/gradle/gradle/issues/26928
-                    problemSpec.category(DefaultProblemCategory.DEPRECATION, TextUtil.screamingSnakeToKebabCase(usage.getType().name()))
+                    problemSpec.category(DEPRECATION, screamingSnakeToKebabCase(usage.getType().name()))
```

Comment:
❓I'm interested if you imported these names in statically, or you were forced by `sC`?

Reply:
I did it intentionally.

Reply:
I am just curious, as I am not a big fan of single-use static imports, especially when the class is not directly in the domain the import is in.